lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -346,6 +346,21 @@ def create_changeset(module, stack_params, cfn):
             result = dict(changed=False, output='ChangeSet %s already exists.' % changeset_name, warnings=[warning])
         else:
             cs = cfn.create_change_set(**stack_params)
+            while True:
+                try:
+                    newcs = cfn.describe_change_set(ChangeSetName=cs['Id'])
+                except Exception as err:
```

Comment:
We should only be catching boto exceptions, not *every* exception here, since there may be issues we aren't able to handle here.

Reply:
Specifically, you'll want to catch botocore.exceptions.BotocoreError here

Reply:
It has been adjusted to only catch BotoCoreError's.